### PR TITLE
do not bomb out if there is no config

### PR DESCRIPTION
**Pull request by kylecrum**

### CODE REVIEW

**Comment by ebenoist** [comment]
Thank you! I've wanted this for awhile.

File: config.go [hunk 1]
```diff
@@ -20,9 +20,8 @@ func initializeConfiguration() {
 	viper.AddConfigPath("$HOME/.rv")
 	var configuration Configuration
 
-	if err := viper.ReadInConfig(); err != nil {
-		log.Fatalf("Error reading config file, %s", err)
-	}
+	_ = viper.ReadInConfig()
```
Comment (ebenoist) [inline config.go:23]: Should we just bail early/safely on an error here?  What happens with the lines below?

Comment (kylecrum) [reply config.go:23]: I ran it locally and it seems like it chugged along just fine.

Comment (ebenoist) [reply config.go:23]: Works for me!

**Pull request merged**